snuba/snapshots/__init__.py

```diff
@@ -41,5 +48,5 @@ def get_descriptor(self) -> SnapshotDescriptor:
 
     @abstractmethod
     @contextmanager
-    def get_table_file(self, table: str) -> Generator[IO[bytes], None, None]:
+    def get_table_file(self, table: str) -> Generator[Iterable[Mapping[str, Any]], None, None]:
```

Comment:
`Mapping[str, Any]` seems like it could use a type alias here so that it's more self-documenting what the intention is here.